tests/scenarios/app-config-environment-test.ts

```diff
@@ -0,0 +1,78 @@
+import merge from 'lodash/merge';
+import { appScenarios } from './scenarios';
+import { PreparedApp } from 'scenario-tester';
+import QUnit from 'qunit';
+
+const { module: Qmodule, test } = QUnit;
+
+appScenarios
+  .only('release')
+  .map('app-config-environment', project => {
+    merge(project.files, {
+      'ember-cli-build.js': `
+        'use strict';
+
+        const EmberApp = require('ember-cli/lib/broccoli/ember-app');
+        const { maybeEmbroider } = require('@embroider/test-setup');
+
+        module.exports = function (defaults) {
+          let app = new EmberApp(defaults, {
+            tests: true,
+            storeConfigInMeta: false,
+          });
+          return maybeEmbroider(app, {});
+        };
+      `,
+      config: {
+        'environment.js': `module.exports = function(environment) {
+          let ENV = {
+            // DEFAULTS
+            modulePrefix: 'my-app',
+            podModulePrefix: '',
+            environment,
+            rootURL: '/',
+            locationType: 'auto',
+            EmberENV: {
+              FEATURES: {
+              },
+              EXTEND_PROTOTYPES: {
+                Date: false
+              }
+            },
+            APP: {},
+
+            // CUSTOM
+            someCustomField: true,
+          };
+          return ENV;
+        };`,
+      },
+      tests: {
+        unit: {
+          'store-config-in-meta-test.js': `
+            import { module, test } from 'qunit';
+            import ENV from 'app-template/config/environment';
+            console.log(ENV.someCustomField)
+
+            module('Unit | storeConfigInMeta', function (hooks) {
+              test('it has loaded the correct config values', async function (assert) {
+                assert.equal(ENV.someCustomField, true);
```

Comment:
This is passing, but the overall `yarn test` run fails with:

```
Built project successfully. Stored in "/var/folders/bf/vdj8yt7533j2c9ryz8mjm8gm0000gp/T/tests-dist-202356-71675-1cft88x.6owb".
    not ok 1 Chrome 113.0 - [undefined ms] - Global error: Uncaught UnrecognizedURLError: /597798672152791/tests/index.html?hidepassed at http://localhost:7357/assets/vendor.js, line 13287
        ---
            browser log: |
                {"type":"log","text":"true"}
                {"type":"error","text":"Uncaught UnrecognizedURLError: /597798672152791/tests/index.html?hidepassed at http://localhost:7357/assets/vendor.js, line 13287\n","testContext":{}}
        ...
    ok 2 Chrome 113.0 - [1 ms] - Unit | storeConfigInMeta: it has loaded the correct config values
    ok 3 Chrome 113.0 - [1 ms] - ember-qunit: Ember.onerror validation: Ember.onerror is functioning properly
```

Have we seen this before? `Global error: Uncaught UnrecognizedURLError:`

Reply:
Discussed in office-hours, my `'environment.js'` override above is missing:
```js
          if (environment === 'test') {
            ENV.locationType = 'none';
          }
```